_plugins/render_author_list.rb

```diff
@@ -9,13 +9,49 @@ def to_author_list(input)
       end.join(', ')
     end
 
+    def find_author_id(author)
+      @context.registers[:site].data["authors"].find { |k, v| v == author}[0]
+    end
+    
     private 
     def get_author_link(author_token)
       url = @context.registers[:site].config['url']
       author = @context.registers[:site].data["authors"][author_token]
       "<a href='#{url}/bios/#{author_token}'>#{author['name']}</a>"
     end
+
+  end
+
+  module PostFilter
+    def filter_by_author(posts, author)
+      filtered_posts = []
+      for post in posts
+        found_author = post["participants"].find { |v| v == author }
+        if found_author
+          filtered_posts.push(post)
+        end
+      end
+      filtered_posts
+    end
+  end
```

Comment:
I think you can just write
`posts.select { |p| p["participants"].include? author } }`